lib/settings_base.py

```diff
@@ -17,9 +17,6 @@
     '.mozilla.net',
 ]
 
-LOG_TABLE_SUFFIX = ''
-EVENT_TABLE_SUFFIX = ''
```

Comment:
This is used in zadmin.models: https://github.com/mozilla/olympia/blob/master/apps/zadmin/models.py#L286